lib/ukio/bookings/handlers/booking_creator.ex

```diff
@@ -20,14 +21,29 @@ defmodule Ukio.Bookings.Handlers.BookingCreator do
     end
   end
 
-  defp generate_booking_data(apartment, check_in, check_out) do
-    %{
-      apartment_id: apartment.id,
-      check_in: check_in,
-      check_out: check_out,
-      monthly_rent: apartment.monthly_price,
-      utilities: 20_000,
-      deposit: 100_000
-    }
+  def generate_booking_data(apartment, check_in, check_out) do
+    if apartment.market == "mars" do
+      %{
+        apartment_id: apartment.id,
+        check_in: check_in,
+        check_out: check_out,
+        monthly_rent: apartment.monthly_price,
+        utilities: calculate_utilities_amount(apartment.market, apartment.square_meters),
+        deposit: apartment.monthly_price
+      }
```

Comment:
At this stage, we only have one market that needs a different treatment: `mars`
We adjust how we generate booking data based on the requirements